visualizations/frontend/other/TTable.vue

```diff
@@ -628,11 +624,22 @@ export default {
               );
             }
           });
-          // remove the default sort direction of columns not sorted by the url
-          sortableColumns.forEach((sc) => delete sc.direction);
           // append remaining sortable but not sorted columns to the configuration
           sortConfig = sortConfig.concat(sortableColumns);
           sortableColumns = sortConfig;
+        } else if(this.defaultSort){
```

Comment:
I am noticing that the formatting is a bit off. There are 4 spaces of indentation instead of 2 in some of the files, and here, the formatter isn't catching to add a space at the correct spot. 


```suggestion
        } else if (this.defaultSort) {
```

Your formatting extensions may need a look at. I can help if you'd like to take a look together.

Reply:
Fixed!